influxdb/1.3/alpine/Dockerfile

```diff
@@ -4,7 +4,7 @@ RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
 
 ENV INFLUXDB_VERSION 1.3.0
 RUN set -ex && \
-    apk add --no-cache --virtual .build-deps wget gnupg tar ca-certificates && \
+    apk add --no-cache --virtual .build-deps wget gnupg tar ca-certificates tzdata && \
```

Comment:
You're going to need to make a new `apk add` for `tzdata` since all of the build dependencies here get removed. See the `kapacitor` and `telegraf` alpine build files for how to do that. `influxdb` doesn't follow that pattern because it doesn't generally make outwards connections that rely on `ca-certificates`.